docs/src/getting-started/cli.md

```diff
@@ -64,8 +64,8 @@ miden-faucet start \
 
 | Option | Description | Default | Required |
 |--------|-------------|---------|----------|
-| `--api-bind-url` | URL to serve the faucet API | - | Yes |
-| `--api-public-url` | Public URL to access the faucet API. If not set, the bind url will be used. | - | No |
+| `--api-port` | Port to bind the API server | - | Yes |
+| `--api-public-url` | Public URL to access the faucet API | - | No |
```

Comment:
nit (feel free to ignore): I wonder if `api-bind-port` would be a bit more self-explanatory.